### ISSUE TITLE
ee-rg get: cat: /tmp/ee-rg.tmp: No such file or directory

### ISSUE THREAD

**Issue by yibie** [eval-exec/eee.el#19]
Every command with "ee-...", error message shows up like this:

> ee-rg get: cat: /tmp/ee-rg.tmp: No such file or directory

My env:

MacBook Pro M1 Pro, macOS 15, emacs 30.0.91.

**Comment by eval-exec** [comment]
Could you enable `ee-debug-message` to `t`, what's the ee related log message inf `*Message*` buffer?

**Comment by yibie** [comment]
ee-yazi opening: cat: /tmp/ee-yazi.tmp: No such file or directory
ee-find-file: cat: /tmp/ee-yazi.tmp: No such file or directory
Use M-x make-directory RET RET to create the directory and its parents

**Comment by eval-exec** [comment]
what's your `eee--load-file-path` variable?

**Comment by yibie** [comment]
/Users/username/.emacs.d/elpaca/builds/eee/eee.elc

**Comment by eval-exec** [comment]
What's the value of `(ee-script-path "eee-rg.sh")`?

**Comment by yibie** [comment]
/Users/username/.emacs.d/elpaca/builds/eee/eee-rg.sh

**Comment by eval-exec** [comment]
can you execute `/Users/username/.emacs.d/elpaca/builds/eee/eee-rg.sh` in your teriminal?
after execution, is there some files has `ee-*` prefix in `/tmp/` dir?

**Comment by eval-exec** [comment]
also, please update `eee.el`'s source code.

**Comment by yibie** [comment]
I can execute the `sh` script in my terminal, and it work well.
After execution, three no `ee-*` preix files in `/tm/` dir.

**Comment by yibie** [comment]
I had updated `eee.el`, now execute `ee-rg`, shows log:
```
wezterm --config enable_wayland=false --config enable_tab_bar=false --config initial_cols=180 --config initial_rows=50 --config window_decorations=\"NONE\" -e bash -c 'cd /Users/username/ && /Users/username.emacs.d/elpaca/builds/eee/eee-yazi.sh  > /tmp/ee-stdout-ee-yazi.tmp'
ee-rg: cd /Users/username/ && /Users/username/.emacs.d/elpaca/builds/eee/eee-rg.sh  > /tmp/ee-stdout-ee-rg.tmp
```
Looks like the command had been executed. But I check the `/tmp/` dir, no `ee-*` files in there still.

**Comment by eval-exec** [comment]
Could you execute `cd /Users/username/ && /Users/username/.emacs.d/
elpaca/builds/eee/eee-rg.sh` in your terminal, what't the output?

**Comment by yibie** [comment]
```
zsh: permission denied: /Users/chenyibin/.emacs.d/
zsh: no such file or directory: elpaca/builds/eee/eee-rg.sh
```

**Comment by eval-exec** [comment]
why you can't `cd /Users/chenyibin/.emacs.d/`?

**Comment by eval-exec** [comment]
> I just forgot to change the path to my actual path, so I encountered an error. After changing it, rg was able to start normally in terminal.

Then ee-rg works?

**Comment by yibie** [comment]
No, just show the message like [before](https://github.com/eval-exec/eee.el/issues/19#issuecomment-2421284543). WezTerm didn't start.

**Comment by eval-exec** [comment]
what's `getenv PATH` in your emacs? Does wezterm in the PATH?

**Comment by yibie** [comment]
After reset the PATH of wezterm, execute `ee-rg`, then wezterm start, and works normally, but there is another problem:

The system windows doesn't focus on wezterm.

**Comment by eval-exec** [comment]
I guess it's related to your window manager's "Focus stealing prevention" related settings.

**Comment by eval-exec** [comment]
I'm living in KDE plasma6, the related setting is:
![image](https://github.com/user-attachments/assets/62fc5ad3-d08c-43ea-a74a-55407d56552a)

**Comment by yibie** [comment]
Thank you, I'm using macOS, the problem isn't relate to you, it's all macOS fault. Let me figure it out.
